contents/docs/api/trends.mdx

```diff
@@ -1,80 +1,320 @@
+            "name": null,
+            "filters": {
+                "events": [
+                    {
+                        "id": "$pageview",
+                        "math": null,
+                        "name": "$pageview",
+                        "type": "events",
+                        "order": 0,
+                        "properties": [],
+                        "math_property": null
+                    }
+                ],
+                "display": "ActionsLineGraph",
+                "filters": [],
+                "insight": "SESSIONS",
+                "session": "avg",
+                "interval": "day",
+                "pagination": {}
+            },
+            "filters_hash": "cache_24b6b973559c99cfffb8a02e3d412ae2",
             "order": null,
-            "name": "$pageview",
-            "math": null,
-            "properties": []
+            "deleted": false,
+            "dashboard": null,
+            "layouts": {},
+            "color": null,
+            "last_refresh": "2021-05-02T05:10:24.682389Z",
+            "refreshing": false,
+            "result": null,
+            "created_at": "2021-04-28T11:29:02.413232Z",
+            "saved": false,
+            "created_by": {
+                "id": 2961,
+                "uuid": "01791342-cc0d-0000-57e5-d8dc4b42cc57",
+                "distinct_id": "AdCojLadUwWOfT0V6qO15URj1spDVKx-c4itZm28QSk",
+                "first_name": "",
+                "email": "test@example.com"
+            }
+        },
+    ]
+}
+```
+
+</HiddenSection>
+
+<Endpoint endpoint='/api/insight/trend' allowedMethods={['get']} />
+
+Get a list of trends from PostHog.
+
+<HiddenSection headingType='h4' title='Query Parameters'>
+
+
+| Parameter | Type / Valid Values | Description | Required |
+| --- | --- | --- | --- |
+| `events` | Array of Events | Basic usage is `[{"id": "$pageview"}]`.<br /><br />For each event, you can specify:<br />- `id` Name of the event<br />- `properties` A Properties object (see below)<br />- `order` (int) order of the event. This is mostly used for funnels | ✅ |
+| `properties` | Array of Properties | The key/values that you want to filter on. Basic usage is: `[{"key": "$browser", "value": "Chrome"}]`<br /><br />For each property, you can specify:<br />- `key` Key of the property<br />- `value` Value you want to filter on<br />- `type` Either `person` or `event`<br />- `operator`, any of: <br />-- `exact`/empty<br />--`is_not`<br />--`icontains`<br />--`not_icontains`<br />--`gt`<br />--`lt`<br />--`is_set` | ❌ |
+| `interval` | String | Interval of the data. `minute`, `hour`, `day`, `week`, `month` | ❌ |
+| `breakdown_value` | String | Split out all entities by any property value or cohort id. | ❌ |
+| `breakdown_type` | String | Default: `property`. Can be set to `cohort`, in which case you can pass through a cohort_id to `breakdown_value` | ❌ |
+| `from_dashboard` | String | | ❌ |
+| `shown_as` | String / `Volume`, and `Stickiness` |  | ❌ |
```

Comment:
If you can only pass specific values to this, no need to specify "String", just pass the values:

- `"volume"`
- `"stickiness"`